packages/api-client-app/src/main/index.ts

```diff
@@ -14,7 +16,7 @@ function createWindow(): void {
     titleBarStyle: 'hidden',
     trafficLightPosition: { x: 11, y: 12 },
     autoHideMenuBar: true,
-    ...(process.platform === 'linux' ? { icon } : {}),
+    icon: iconPath,
```

Comment:
If that’s not set, it’s using the app icon. Let’s not change it, until we need to. :v: